sql/dql/basic-queries/sql-practice-order-results.md

```diff
@@ -23,11 +23,12 @@ standards:
   sql.read-single-table.3: 1000
 
 links:
-  - '[MySQL ORDER BY documentation](https://dev.mysql.com/doc/refman/5.7/en/order-by-optimization.html){website}'
+  - '[MySQL ORDER BY documentation](https://dev.mysql.com/doc/refman/5.7/en/order-by-optimization.html){documentation}'
   - '[blog post tutorial on SELECT](http://www.dofactory.com/sql/select){website}'
   - '[video tutorial on MySQL SELECT](https://www.youtube.com/watch?v=BgK88mlgA6I){video}'
 
 ---
+
```

Comment:
newlines above section name are not needed btw
imo it looks better without them